build/params_debug.go

```diff
@@ -1,11 +0,0 @@
-//go:build debug
-// +build debug
-
-package build
-
-func init() {
-	InsecurePoStValidation = true
-	BuildType |= BuildDebug
-}
-
-// NOTE: Also includes settings from params_2k
```

Comment:
oo, good point.


I was hoping debug networks could be a runtime rather than build time decision. But there are some things that aren't ready for that, and I don't really want to make this PR any bigger than it already is.

I've added a params_debug back in. It sets the SlashablePowerDelay, Insecure PoSt, and sets the "BuildType" to debug.